include/swift/AST/Decl.h

```diff
@@ -4874,6 +4874,11 @@ class EnumDecl final : public NominalTypeDecl {
     return getAttrs().hasAttribute<IndirectAttr>();
   }
 
+  /// True if the enum is marked with '@cdecl'.
+  bool isCDeclEnum() const {
```

Comment:
There are 4 places where we're checking "isObjC or isCDeclEnum". Can we coalesce those into a single "is C-compatible enum" predicate?

Reply:
Good idea, I added a `isCCompatibleEnum` for this. Most of the implementation is shared between `@cdecl` and `@objc` enums so most call sites use this new service.